packages/trader/src/sass/app/modules/reports.scss

```diff
@@ -777,11 +774,6 @@ $side-padding: 1.2em;
             font-weight: bold;
             white-space: normal;
         }
-        .table__cell {
```

Comment:
any reason for removing this?

Reply:
table content shouldn't be bold and there weren't any padding top/bottom to override